components/admin_console/admin_sidebar/index.js

```diff
@@ -19,6 +26,10 @@ function mapStateToProps(state) {
     const siteName = config.SiteName;
     const adminDefinition = getAdminDefinition(state);
     const consoleAccess = getConsoleAccess(state);
+    const taskListStatus = getBool(state, OnboardingTaskCategory, OnboardingTaskList.ONBOARDING_TASK_LIST_SHOW);
+    const isUserFirstAdmin = isFirstAdmin(state);
+    const isMobileView = isMobile();
+    const showTaskList = isUserFirstAdmin && taskListStatus && !isMobileView;
```

Comment:
I'm wondering if we could not make a selector out of this 🤔

Reply:
whereas sound tempting, I think is not worth creating a selector of selectors. Also, Im not sure if it could be performance costing to move this imports to a central place.